File: Exam Preparation (RetakeExam - 5 September)/JavaScript Blog/controllers/anime.js
const Anime = require('../models/Anime');

module.exports = {
	index: (req, res) => {
        Anime.find().then(animes => {
            res.render('anime/index', {'animes': animes})
        });
	},
	createGet: (req, res) => {
        res.render('anime/create');
	},
	createPost: (req, res) => {
        let animeArgs = req.body;
        if (!animeArgs.rating || !animeArgs.name || !animeArgs.description || !animeArgs.watched){
            res.redirect('/'); 
            return;
        }
        Anime.create(animeArgs).then(anime => {
            res.redirect('/'); 
        });
    },
	deleteGet: (req, res) => {
        let id = req.params.id;
        Anime.findById(id).then(anime => {
            if(!anime){
                res.redirect('/');
                return;
            }
            res.render('anime/delete', anime);
        })
	},
	deletePost: (req, res) => {
        let id = req.params.id;
        Anime.findByIdAndRemove(id).then(anime => {
            res.redirect('/');
        })
    }
};

File: Exam Preparation (RetakeExam - 5 September)/C# Blog/AnimeList/Models/Anime.cs
using System.ComponentModel.DataAnnotations;
using System.Web.Mvc;

namespace AnimeList.Models
{
    public class Anime
    {
        public int Id { get; set; }

        [Required]
        public int Rating { get; set; }

        [Required]
        public string Name { get; set; }

        [Required]
        public string Description { get; set; }

        [Required]
        public string Watched { get; set; }
    }
}